scripts/data/generate_data.py

```diff
@@ -233,12 +195,12 @@ def format_outpoint(previous_output, median_time_past):
             "cached": False,
         },
         "block_height": int(previous_output["block_height"]),
-        "median_time_past": median_time_past,
+        "median_time_past": int[previous_output["median_time_past"]],
```

Comment:
it should be `previous_output["median_timestamp"]`

Reply:
It depends on whether MTP comes from RPC or bigquery file
Ok